### PR TITLE
Fix validation about zip slip

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

I made a mistake in #1145. This PR fixes it.

### Related Issue:

Nothing

### How Has This Been Tested?:

Nothing

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [X] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [x] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: pkg/agent/sidecar/service/restorer/restorer.go [hunk 1]
```diff
@@ -212,6 +212,11 @@ func (r *restorer) restore(ctx context.Context) (err error) {
 
 		log.Debug("restoring: ", target)
 
+		if strings.Contains(target, "..") {
+			log.Warn(errors.ErrPathNotAllowed(target))
+			continue
```
Comment (rinx) [inline pkg/agent/sidecar/service/restorer/restorer.go:217]: Just skipping the invalid file. is it okay?

Comment (kpango) [reply pkg/agent/sidecar/service/restorer/restorer.go:217]: I think it's okay, because tr traverses directory we can skip upper directory

Comment (rinx) [reply pkg/agent/sidecar/service/restorer/restorer.go:217]: thanks

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**